<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="config_tab">
         <attribute name="title">
          <string>Config</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="7" column="0">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Manual reading (Input coordinates separated by commas in tuples, ex. &quot;(20,30,40),(10,100,20)&quot;)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="manual_reading">
              <property name="text">
               <string>Manual</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPlainTextEdit" name="manual_positions">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="11" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Z points</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spin_x_points">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spin_y_points">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="spin_total_points">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>999999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Total points</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Number of points</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>X points</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Y points</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spin_z_points">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="13" column="0">
           <widget class="QPushButton" name="button_generate_program_file">
            <property name="text">
             <string>Generate Movement File</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="3" column="1">
             <widget class="QCheckBox" name="check_box_y">
              <property name="text">
               <string>Y axis</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Planes</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="check_box_x">
              <property name="text">
               <string>X axis</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="check_box_z">
              <property name="text">
               <string>Z axis</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="10" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sensor_tab">
         <attribute name="title">
          <string>Sensor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Select file</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Read file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="1" column="1">
             <widget class="QComboBox" name="combo_box_sensor_type">
              <item>
               <property name="text">
                <string>Analog</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>I2C</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SPI</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Other</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Sensor</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Sensor type</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="generate_sensor_file">
            <property name="text">
             <string>Generate sensor file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="view_tab">
         <attribute name="title">
          <string>View</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0" colspan="3">
           <widget class="QOpenGLWidget" name="graphic"/>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="graph_file_path"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="graph_load_file">
            <property name="text">
             <string>Load file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="graph_select_file">
            <property name="text">
             <string>Select file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
